disnake/interactions/base.py

```diff
@@ -858,6 +864,12 @@ async def send_message(
             before deleting the message we just sent. If the deletion fails,
             then it is silently ignored.
 
+            Can be up to 15 minutes, after which the interaction token expires
```

Comment:
up to 15 minutes from the creation of the interaction***

Reply:
https://github.com/DisnakeDev/disnake/pull/816/commits/23ed979d5aaf1f6d3a19ae8d552e3fc5a3108446 something like this?

Reply:
yeah, i think so